src/Content/Text/BBCode.php

```diff
@@ -578,7 +578,10 @@ private static function convertAttachment($return, $simplehtml = false, $tryoemb
 					$return .= sprintf('<a href="%s" target="_blank"><img src="%s" alt="" title="%s" class="attachment-preview" /></a><br />', $data["url"], self::proxyUrl($data["preview"], $simplehtml), $data["title"]);
 				}
 
-				if (!empty($data['title']) && !empty($data['url'])) {
+				// Show a picture only when the BBCode is meant for posts to connector networks
+				if (($simplehtml != 0) && ($data["type"] == "photo") && !empty($data["url"]) && !empty($data["image"])) {
```

Comment:
Please add this conditional to line 575 instead, or there's a risk of duplicated image again for non-connector posts.